activerecord/lib/active_record/connection_adapters/sqlite3/schema_definitions.rb

```diff
@@ -0,0 +1,23 @@
+module ActiveRecord
+  module ConnectionAdapters
+    module SQLite3
+      module ColumnMethods
+        def primary_key(name, type = :primary_key, **options)
+          if options.delete(:auto_increment) == true && type == :integer
```

Comment:
Here too, though because SQLite Is Special, I think we can just treat int and bigint identically?